src/pointer-analysis/value_set_dereference.cpp

```diff
@@ -454,7 +454,13 @@ value_set_dereferencet::valuet value_set_dereferencet::build_reference_to(
     // This is stuff like *((char *)5).
     // This is turned into an access to __CPROVER_memory[...].
 
-    const symbolt &memory_symbol=ns.lookup(CPROVER_PREFIX "memory");
+    if(language_mode==ID_java)
+    {
+      result.value=nil_exprt();
+      return result;
+    }
+    
+    const symbolt &memory_symbol=ns.lookup(CPROVER_PREFIX "memory");    
```

Comment:
There appears to be trailing whitespace (otherwise that line should not be flagged as modified).